server/models/User.js

```diff
@@ -16,7 +16,7 @@ const UserSchema = new Schema({
     },
     fullname: String, 
     discord: String,
-    pinnedPosts: { type: [ObjectId], default: [] },
+    pinnedPosts: { type: [String], default: [] },
```

Comment:
Addressing your comment about any benefits to it being `[ObjectId]`: tbh Im not too sure lol. 
When you tested it with Postman, did you create a post and use that same id as a string to fetch it back/update it? If thats the case and it works, then should be fine.

I made it ObjectId initially because I didn't know if mongo generated id's would work the same way if they were a string.

Reply:
yeah I used the id as a string and everything worked ok, I think both String and ObjectId are acceptable but I thought it would be easier to keep things as strings for compatibility haha.